### PR TITLE
Change -disable-wallet-api to -enable-wallet-api

### LINKED ISSUE #1178: Change -disable-wallet-api to -enable-wallet-api
Use `-enable-wallet-api` by default in `run.sh` and in electron.

Add a new `make run-wallet` command, this command will add all flags necessary for running a client.

Change `make run` to add no flags, and disable browser launching.

### PR DESCRIPTION
**Pull request by gz-c**
Fixes #1178 

Changes:
- Change `-disable-wallet-api` to `-enable-wallet-api` and disable by default

Does this change need to mentioned in CHANGELOG.md?
Yes

### CODE REVIEW

File: src/daemon/gateway.go [hunk 1]
```diff
@@ -18,15 +18,15 @@ import (
...
+	EnableWalletAPI bool
 }
 
 // NewGatewayConfig create and init an GatewayConfig
 func NewGatewayConfig() GatewayConfig {
 	return GatewayConfig{
-		BufferSize:       32,
-		DisableWalletAPI: false,
+		BufferSize:      32,
+		EnableWalletAPI: false,
```
Comment (iketheadore) [inline src/daemon/gateway.go]: EnableWalletAPI: true?

Comment (gz-c) [reply src/daemon/gateway.go]: It's disabled by default now, we enable it through run.sh and electron-main.js
The default settings should be the normal settings for running a server node

**Comment by gz-c** [comment]
@iketheadore rebased, please re-check

File: CHANGELOG.md [hunk 2]
```diff
@@ -26,12 +26,16 @@ and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.
 
 - #1168, Updated CLI README
 - `-launch-browser` is set to false by default
+<<<<<<< HEAD
```
Comment (iketheadore) [inline CHANGELOG.md]: Conflicts not fixed here.

File: CHANGELOG.md [hunk 3]
```diff
@@ -26,12 +26,16 @@ and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.
...
 - `/wallet` API endpoint, remove sensitive data from the response, and fix the data format to be the same as `/wallet/create`
 - `/wallets` API endpoint, remove sensitive data from the response
 - `/wallet/create` API endpoint, add `encrypt(bool)` and `password` argument
 - `/wallet/newAddress` API endpoint, add `password` argument
 - `/wallet/spend` API endpoint, add `password` argument
 
+=======
+- Change `-disable-wallet-api` to `-enable-wallet-api`, and disable the wallet API by default
+>>>>>>> Change -disable-wallet-api to -enable-wallet-api
```
Comment (iketheadore) [inline CHANGELOG.md]: And here

File: CHANGELOG.md (hunk 2 above)
Comment (gz-c) [reply CHANGELOG.md]: fixed changelog conflicts

**Pull request merged**